docs/content/routing/providers/docker.md

```diff
@@ -149,18 +152,18 @@ and the router automatically gets a rule defined by `defaultRule` (if no rule fo
 
 --8<-- "content/routing/providers/service-by-label.md"
 
-??? example "Automatic service assignment with labels"
+??? example "Automatic assignment with one Service"
 
     With labels in a compose file
 
     ```yaml
     labels:
       - "traefik.http.routers.myproxy.rule=Host(`example.net`)"
-      # service myservice gets automatically assigned to router myproxy
+      # No link specified, and yet myproxy router is linked to myservice
```

Comment:
I'm not sure about the improvement brought by the new wording here.

Reply:
Yes, I agree. I removed it.